<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mainPageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Virtual Machine - Network Management </string>
    </property>
   </widget>
   <widget class="QListWidget" name="connectedList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>256</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="connectedLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>281</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Interfaces connected to {vmName} :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="otherLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>311</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Other interfaces :</string>
    </property>
   </widget>
   <widget class="QListWidget" name="otherList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>256</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
